Compile-time warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]

./third_party/gemmlowp/public/../internal/multi_thread_gemm.h:368:38: warning: implicit conversion loses integer precision: 'unsigned long' to 'int' [-Wshorten-64-to-32]
    int workers_count = tasks.size() - 1;
        ~~~~~~~~~~~~~   ~~~~~~~~~~~~~^~~
1 warning generated.

See https://github.com/google/gemmlowp/blob/master/internal/multi_thread_gemm.h#L368.

--- a/internal/multi_thread_gemm.h
+++ b/internal/multi_thread_gemm.h
@@ -365,7 +365,7 @@ class WorkersPool {
   void Execute(const std::vector<Task*>& tasks) {
     assert(tasks.size() >= 1);
     // One of the tasks will be run on the current thread.
-    unsigned long workers_count = tasks.size() - 1;
+    std::size_t workers_count = tasks.size() - 1;
     CreateWorkers(workers_count);
     assert(workers_count <= workers_.size());
     counter_to_decrement_when_ready_.Reset(workers_count);

--- a/internal/multi_thread_gemm.h
+++ b/internal/multi_thread_gemm.h
@@ -365,7 +365,7 @@ class WorkersPool {
   void Execute(const std::vector<Task*>& tasks) {
     assert(tasks.size() >= 1);
     // One of the tasks will be run on the current thread.
-    int workers_count = tasks.size() - 1;
+    unsigned long workers_count = tasks.size() - 1;
     CreateWorkers(workers_count);
     assert(workers_count <= workers_.size());
     counter_to_decrement_when_ready_.Reset(workers_count);

